liblangutil/Token.h

```diff
@@ -322,6 +324,22 @@ namespace TokenTraits
 			tok == Token::Default || tok == Token::For || tok == Token::Break || tok == Token::Continue || tok == Token::Leave ||
 			tok == Token::TrueLiteral || tok == Token::FalseLiteral || tok == Token::HexStringLiteral || tok == Token::Hex;
 	}
+	constexpr bool isExperimentalSolidityKeyword(Token tok)
```

Comment:
Aren't these tokens keywords in both experimental and non-experimental Solidity? Seeing this name in isolation, I'd assume it's experimental only.

```suggestion
	constexpr bool isSolidityKeyword(Token tok)
```

Reply:
No, those are all keywords of experimental solidity (including all keywords of experimental solidity that are also keywords of normal solidity), so I'd say the name is fitting.
We can later on implement it as a disjunction of the shared keywords and ``isExperimentalSolidityOnlyKeyword``, though.

Reply:
Ah, ok.